_targets/.gitignore

```diff
@@ -0,0 +1,5 @@
+*
+!.gitignore
+!meta
+meta/*
+!meta/meta
```

Comment:
This extra `gitignore` is not needed. I typically ignore everything in `_targets/`. Before merging delete these files so they don't go to github.